packages/blockchain-extension/extension/fabric/FabricEnvironment.ts

```diff
@@ -55,10 +55,13 @@ export class FabricEnvironment extends EventEmitter {
     public async getNodes(withoutIdentities: boolean = false): Promise<FabricNode[]> {
         const rootNodesPath: string = path.resolve(this.path, 'nodes');
         const nodes: FabricNode[] = await this.loadNodes(rootNodesPath);
+        const visibleNodes: FabricNode[] = nodes.filter((_node: FabricNode) => {
```

Comment:
do we need to pass into this function whether you want all the nodes or just the visible ones?

Reply:
Good call, we will probably need it in the future. Will implement that.